TypeCobol.Analysis/Cfg/ControlFlowGraphBuilder.PerformTarget.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System.Collections.Generic;
+using TypeCobol.Compiler.CodeElements;
+using TypeCobol.Compiler.Nodes;
+using SectionNode = TypeCobol.Compiler.Nodes.Section;
+
+namespace TypeCobol.Analysis.Cfg
+{
+    public partial class ControlFlowGraphBuilder<D>
+    {
+        /// <summary>
+        /// PerfomrTarget class to capture target sentences and procedures from a PERFORM instruction.
+        /// </summary>
+        private class PerformTarget
+        {
+            /// <summary>
+            /// All target sentences
+            /// </summary>
+            internal List<Sentence> Sentences { get; private set; }
+            /// <summary>
+            /// All target procedures.
+            /// </summary>
+            internal List<Procedure> Procedures { get; private set; }
+
+            public PerformTarget(List<Sentence> sentences, List<Procedure> procedures)
+            {
+                this.Sentences = sentences;
+                this.Procedures = procedures;
+            }
+        }
+
+        /// <summary>
+        /// Cache of PerformTarget instances already calculated for a Perform procedure 
+        /// </summary>
+        private Dictionary<PerformProcedure, PerformTarget> _performTargetCache;
+        private PerformTarget ComputePerformTarget(PerformProcedure p, SectionNode sectionNode)
```

Comment:
naming, this is more like `GetPerformTarget` since we do not compute the target each time the method is called.

Reply:
done in 4b969b9